<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt1Class</class>
 <widget class="QWidget" name="qt1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt1</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(170, 170, 127) url(:/img/bbb.png) no-repeat center center;
</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>300</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(12,32,43,.8);
width: 300px;
height: 200px;
color: #fff;
border: 3px solid red;
border-radius: 50%;
font-size:40px;</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true">#tab{
	background: rgb(170, 170, 127);
}</string>
      </property>
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="styleSheet">
          <string notr="true">#addBtn {

color: red;
background: rgba(23,32,43,.5)
}</string>
         </property>
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">QHeaderView                    
{
    background:blue;       
}


QHeaderView::section           
{
    font-size:14px;                
    font-family:&quot;Microsoft YaHei&quot;; 
    color:#FFFFFF;                 
 
    background:#60669B;            
    border:none;                  
    text-align:left;               
 
 
    min-height:49px;               
    max-height:49px;               
 
    margin-left:0px;              
    padding-left:0px;              
}</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>页</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="styleSheet">
          <string notr="true">QHeaderView                    
{
    background:blue;     
}


QHeaderView::section           
{
        font-size:20px;                
    font-family:&quot;Microsoft YaHei&quot;; 
 	   color:#FFFFFF;             
 background:#60669B;                     
      text-align:top;           
   	height: 50px;  
	border: 1px solid #adf;            
 
    margin-left:20px;              
    padding-left:20px; 
	border-color:  red;   
    border-right:node;
}


QTableWidget              
{
	qproperty-showGrid: false;
qproperty-gridStyle: 1px solid red;
    background:green;                
 
    font-size:30px;               
    font-family:&quot;Microsoft YaHei&quot;;
    color:#aaa;
	border-right: 1px solid red;                
}


QTableWidget::item                
{
    border: 1px solid red; 
	border-right: none;
}
</string>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>wod </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>OK</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>D</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="3" column="4">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item row="4" column="4">
          <property name="text">
           <string>23</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt1.qrc"/>
 </resources>
 <connections/>
</ui>
